client/state/notices/test/selectors.js

```diff
@@ -20,4 +20,44 @@ describe( 'selectors', () => {
 			expect( notices ).toEqual( [ { noticeId: 1 }, { noticeId: 2 }, { noticeId: 3 } ] );
 		} );
 	} );
+
+	describe( 'getNotice()', () => {
```

Comment:
Thanks for adding tests! Looks like these are wonderful candidates for a separate PR.